W-13130406 Reword sentence (#828)
mulesoft/docs-gateway: gateway/1.9/modules/ROOT/pages/index.adoc

** {empty}
include::partial$prerequisites.adoc[tag=ubuntu]

> For protecting a small number of APIs with low transaction volume where high availability and robustness aren't critical, use a standalone gateway. Otherwise, use an Ingress controller in Kubernetes.

Author: ```suggestion
If you are protecting a small number of APIs with low transaction volume, and high availability and robustness are not a concern, consider using a standalone gateway. Otherwise, use an Ingress controller in Kubernetes.
```

> If you are protecting a small number of APIs with low transaction volume, and high availability and robustness are not a major concern, consider using a standalone gateway. Otherwise, use an Ingress controller in Kubernetes.

Reviewer: ```suggestion
If you are protecting a small number of APIs with low transaction volume and high availability and robustness are not a major concern, consider using a standalone gateway. Otherwise, use an Ingress controller in Kubernetes.
```

Reviewer: Honestly not sure, I imagine you know better than me.

Author: I'm rewriting it because you may be right about the commas (or not -- can't we use them for parenthetical phrases?) but it's obviously not very readable or we'd know.
Here's a new version:
_A standalone gateway is suitable for protecting a limited number of APIs with low transaction volume. However, for other scenarios, or if high availability and reliability are critical, consider using an Ingress controller in Kubernetes._
